test/TensorFlow/control_flow_objc.swift

```diff
@@ -0,0 +1,30 @@
+// RUN: %target-swift-frontend -Xllvm -tf-dump-intermediates -Xllvm -tf-dump-graph -O -emit-sil %s -verify | %FileCheck %s
+// REQUIRES: OS=macosx
```

Comment:
awesome!

we don't have CI for mac tests yet, but that's coming up.
@dan-zheng as FYI.

one question: does this compiler-only test run on Linux as well?

Reply:
Good point. The compile-time test cases are merged into a single file.